PublicAPI/Microsoft.ApplicationInsights.dll/net45/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,59 @@
+Microsoft.ApplicationInsights.DataContracts.PageViewPerformanceTelemetry.ItemTypeFlag.get -> Microsoft.ApplicationInsights.DataContracts.SamplingTelemetryItemTypes
+Microsoft.ApplicationInsights.DataContracts.PageViewTelemetry.ItemTypeFlag.get -> Microsoft.ApplicationInsights.DataContracts.SamplingTelemetryItemTypes
+Microsoft.ApplicationInsights.DataContracts.RequestTelemetry.ItemTypeFlag.get -> Microsoft.ApplicationInsights.DataContracts.SamplingTelemetryItemTypes
+Microsoft.ApplicationInsights.DataContracts.ISupportAdvancedSampling.IsProactivelySampledOut.get -> bool
+Microsoft.ApplicationInsights.DataContracts.ISupportAdvancedSampling.IsProactivelySampledOut.set -> void
+Microsoft.ApplicationInsights.DataContracts.DependencyTelemetry.IsProactivelySampledOut.get -> bool
+Microsoft.ApplicationInsights.DataContracts.EventTelemetry.IsProactivelySampledOut.get -> bool
+Microsoft.ApplicationInsights.DataContracts.TraceTelemetry.IsProactivelySampledOut.get -> bool
+Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.IsProactivelySampledOut.get -> bool
+Microsoft.ApplicationInsights.DataContracts.PageViewPerformanceTelemetry.IsProactivelySampledOut.get -> bool
+Microsoft.ApplicationInsights.DataContracts.PageViewTelemetry.IsProactivelySampledOut.get -> bool
+Microsoft.ApplicationInsights.DataContracts.RequestTelemetry.IsProactivelySampledOut.get -> bool
+Microsoft.ApplicationInsights.DataContracts.DependencyTelemetry.IsProactivelySampledOut.set -> void
+Microsoft.ApplicationInsights.DataContracts.EventTelemetry.IsProactivelySampledOut.set -> void
+Microsoft.ApplicationInsights.DataContracts.TraceTelemetry.IsProactivelySampledOut.set -> void
+Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.IsProactivelySampledOut.set -> void
+Microsoft.ApplicationInsights.DataContracts.PageViewPerformanceTelemetry.IsProactivelySampledOut.set -> void
+Microsoft.ApplicationInsights.DataContracts.PageViewTelemetry.IsProactivelySampledOut.set -> void
+Microsoft.ApplicationInsights.DataContracts.RequestTelemetry.IsProactivelySampledOut.set -> void
+Microsoft.ApplicationInsights.DataContracts.ISupportAdvancedSampling.SupportsProactiveSampling.get -> bool
+Microsoft.ApplicationInsights.DataContracts.ISupportAdvancedSampling.SupportsProactiveSampling.set -> void
+Microsoft.ApplicationInsights.DataContracts.DependencyTelemetry.SupportsProactiveSampling.get -> bool
+Microsoft.ApplicationInsights.DataContracts.EventTelemetry.SupportsProactiveSampling.get -> bool
+Microsoft.ApplicationInsights.DataContracts.TraceTelemetry.SupportsProactiveSampling.get -> bool
+Microsoft.ApplicationInsights.DataContracts.ExceptionTelemetry.SupportsProactiveSampling.get -> bool
+Microsoft.ApplicationInsights.DataContracts.PageViewPerformanceTelemetry.SupportsProactiveSampling.get -> bool
+Microsoft.ApplicationInsights.DataContracts.PageViewTelemetry.SupportsProactiveSampling.get -> bool
```

Comment:
Proactive -> Head. It's how it's typically called (head based sampling vs. tail based sampling)

Reply:
It does work nicely with `SupportsHeadSampling`, but it does not play well with the following:
- `IsProactivelySampledOut`? -> `IsHeadSampled`? -> `IsSampledOutAtHead`?